docs/ingestion/input-sources.md

```diff
@@ -1063,7 +1063,7 @@ The following is a sample spec for a S3 warehouse source:
 
 ### Catalog Object
 
-The catalog object supports `local` and `hive` catalog types.
+The catalog object supports `local`,`hive` and `rest` catalog types.
```

Comment:
Nit: maybe reorder the catalog sections by relevance/usage — rest, hive, and local?